easystacks/software.eessi.io/2023.06/eessi-2023.06-eb-4.8.2-system.yml

```
@@ -2,3 +2,4 @@ easyconfigs:
   - EasyBuild-4.8.2.eb:
       options:
         from-pr: 19105
+  - Nextflow-23.10.0.eb
```

This easyconfig was not part of the 4.8.2 release, so we need `from-pr`:
```suggestion
  - Nextflow-23.10.0.eb:
      options:
        from-pr: 19172
```

---

Argh, protected branch, I'll close this and open a new PR from my fork